roxie/udplib/udptrr.cpp

```diff
@@ -706,6 +679,8 @@ class CReceiveManager : implements IReceiveManager, public CInterface
     typedef std::map<ruid_t, CMessageCollator*> uid_map;
     uid_map         collators;
     SpinLock collatorsLock; // protects access to collators map
+
+#ifdef _USE_INFLIGHT
```

Comment:
You may as well remove this code (rather than conditional compile) if you are also removing the code that updated it,

Reply:
ok, wasn't sure if we wanted to keep it for any future work.